Happy Dance/Assets/_Scripts/GameTimer.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections;
+
+using UnityEngine;
+using UnityEngine.Assertions;
+using UnityEngine.Playables;
+
+using Liminal.Core.Fader;
+using Liminal.SDK.Core;
+
+/// <summary>
+/// GameTimer is sued to control the ending of the happy dance experience.
```

Comment:
GameTimer > GameController as it's really is controlling the flow of the game. Or AppController is fine too.

Also there's a typo, sued > used

Reply:
Done